docs/reference/modules/node.asciidoc

```diff
@@ -89,93 +91,85 @@ memory and CPU in order to deal with the gather phase.
+by configuring three of your nodes to be dedicated control nodes.
+Dedicated control nodes only have the `control` role, allowing them to
+focus on cluster management tasks. While control nodes can also behave as
 <<coordinating-node,coordinating nodes>> and route search and indexing requests
-from clients to data nodes, it is better _not_ to use dedicated master nodes for
+from clients to data nodes, it is better _not_ to use dedicated control nodes for
 this purpose.
 
-To create a dedicated master-eligible node, set:
+To create a dedicated control node, set:
 
 [source,yaml]
 -------------------
-node.roles: [ master ]
+node.roles: [ control ]
 -------------------
 
 [[voting-only-node]]
-===== Voting-only master-eligible node
+===== Voting-only control nodes
 
-A voting-only master-eligible node is a node that participates in
-<<modules-discovery,master elections>> but which will not act as the cluster's
-elected master node. In particular, a voting-only node can serve as a tiebreaker
+A voting-only control node participates in the
+<<modules-discovery,election process>> and cluster state publishing, 
+but can never become the manager node. 
+In particular, a voting-only node can serve as a tiebreaker
 in elections.
 
-It may seem confusing to use the term "master-eligible" to describe a
-voting-only node since such a node is not actually eligible to become the master
-at all. This terminology is an unfortunate consequence of history:
-master-eligible nodes are those nodes that participate in elections and perform
-certain tasks during cluster state publications, and voting-only nodes have the
-same responsibilities even if they can never become the elected master.
-
-To configure a master-eligible node as a voting-only node, include `master` and
-`voting_only` in the list of roles. For example to create a voting-only data
+To configure a control node as a voting-only node, include `control` and
+`voting_only` in its list of roles. For example to create a voting-only data
 node:
 
 [source,yaml]
 -------------------
-node.roles: [ data, master, voting_only ]
+node.roles: [ data, control, voting_only ]
 -------------------
 
 IMPORTANT: The `voting_only` role requires the {default-dist} of {es} and is not
 supported in the {oss-dist}. If you use the {oss-dist} and add the `voting_only`
 role then the node will fail to start.  Also note that only nodes with the
-`master` role can be marked as having the `voting_only` role.
+`control` role can have the `voting_only` role.
 
-High availability (HA) clusters require at least three master-eligible nodes, at
-least two of which are not voting-only nodes. Such a cluster will be able to
-elect a master node even if one of the nodes fails.
+High availability (HA) clusters require at least three control nodes, at
+least two of which are not voting-only nodes. This enables a cluster to
+promote a control node to manager even if one of the control nodes fails.
```

Comment:
```suggestion
elect a control node to manager even if the current manager node fails.
```

Reply:
Was sort of "trying this on" as an alternative to elect. I like the idea of select/selection.